src/content/structured/components/skeleton/guidance.mdx

```diff
@@ -46,15 +46,15 @@ Use the circle variant as a placeholder for any circular components, like avatar
 
 ### Text
 
-Use the text variant as a placeholder for any text or labels which sit outside of any component. This applies to helper text for input labels, headings, and body text.
+Use the text variant as a placeholder for any text or labels that sit outside of any component. This applies to helper text for input labels, headings and body text.
 
 ### Rectangle
 
 Use the rectangle variant as a placeholder for any other components. This includes buttons, chips and tags, input fields and cards.
 
 ## When to use
 
-Use skeleton variants as a placeholder for when a full page, or individual components are loading. Don't nest skeletons. If a label or text lives inside a component, for example in cards, only use the skeleton variant of the component which houses the other components.
+Use skeleton variants as a placeholder for when a full page or individual components are loading. Don't nest skeletons. If a label or text lives inside a component, for example in cards, only use the skeleton variant of the component that houses the other components.
```

Comment:
```suggestion
Use skeleton variants as a placeholder for when a full page or individual components are loading. Don't nest skeletons. If a label or text is wrapped inside a component i.e. in cards, only use the skeleton variant of the component that wraps the other components.
```
Changed some wording slightly...think there was a slight pun originally there with 'skeleton' and 'lives' 🤭 

Have added more technical language.